src/api/subscriptions.js

```diff
@@ -145,7 +145,7 @@ export class Subscriptions {
  /**
   * Starts the save subscriptions flow.
   * @param {!SaveSubscriptionRequest} request
-  * @return {!Promise}
+  * @return {!Promise<!boolean>} status or promise of status of request
```

Comment:
No need for `!` in `!boolean` - it's implied.

Reply:
Done